views/default/input/access.php

```diff
@@ -66,7 +66,7 @@
 }
 
 // don't call get_default_access() unless we need it
-if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
+if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
```

Comment:
isset's additional logic of checking !null I think adds too much confusion in this case.